infra/build/functions/build_and_run_coverage.py

```diff
@@ -223,6 +243,110 @@ def get_build_steps(  # pylint: disable=too-many-locals, too-many-arguments
       build_lib.http_upload_step(latest_report_info_body,
                                  latest_report_info_url,
                                  LATEST_REPORT_INFO_CONTENT_TYPE))
+
+  #currently fuzz introspector only supports c and c++
+  if project.fuzzing_language in ['c', 'c++']:
+    #removes index.html from the end of url
+    coverage_url = bucket.html_report_url[:-11]
+    build_steps.extend(
+        get_fuzz_introspector_steps(project, project_name, base_images_project,
+                                    config, coverage_url))
+  return build_steps
+
+
+def get_fuzz_introspector_steps(project, project_name, base_images_project,
+                                config, coverage_url):
+  build_steps = []
+  FI_dir = '/workspace/fuzz-introspector/'
+  oss_integration_dir = 'oss_fuzz_integration/'
+
+  report_date = build_project.get_datetime_now().strftime('%Y%m%d')
+  bucket = IntrospectorBucket(project.name, report_date, PLATFORM,
+                              config.testing)
+
+  build = build_project.Build('libfuzzer', 'instrumentor', 'x86_64')
+  env = build_project.get_env(project.fuzzing_language, build)
+
+  clone_step = {
+      'args': [
+          'clone', 'https://github.com/ossf/fuzz-introspector.git', '--depth',
```

Comment:
@oliverchang Do we need to pin this?
Totally happy to ignore this for now.

Reply:
Yes, it would've been a good idea to pin this to the same hash used for the clang build. That said, we don't need this anymore after https://github.com/google/oss-fuzz/pull/7122 right?

Reply:
yes, won't need these steps.